srcpkgs/calls/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'calls'
+pkgname=calls
+version=0.1.7
+revision=1
+wrksrc=${pkgname}-v${version}
+build_style=meson
+configure_args="-Dtests=false"
```

Comment:
Why? Do the tests fail, or is this just to reduce build time? We do want tests to work.